mypyc/primitives/dict_ops.py

```diff
@@ -108,6 +108,19 @@ def emit_new_dict(emitter: EmitterInterface, args: List[str], dest: str) -> None
     error_kind=ERR_MAGIC,
     emit=emit_new_dict)
 
+dict_new_op = c_custom_op(
```

Comment:
For consistency with other ops, add a single-line comment (e.g. "Create an empty dictionary").

Can we remove the old op, or is it still needed for something?

Reply:
The old op is actually a miss, it should be removed.